std_msgs/Header header
geometry_msgs/Pose2D pose
int32 exception
float32 weight